spec/factories/auth_source_factory.rb

```diff
@@ -36,4 +36,8 @@
     port 225  # a reserved port, should not be in use
     attr_login 'uid'
   end
+
+  factory :dummy_auth_source, class: DummyAuthSource do
+    name 'DerpLAP'
```

Comment:
I'm all for easier auth source development, but there's the requirement for testing a mocked source in the specs and then some dummy auth source for development. The latter could also easily be extracted as a plugin.

The sole problem / question for me is: Can we deactivate the auth source by default, so that actual users of OP aren't confused by that option.